packages/opentelemetry-types/tsconfig.json

```diff
@@ -6,5 +6,11 @@
   },
   "include": [
     "src/**/*.ts"
-  ]
+  ],
+  "typedocOptions": {
+    "name": "OpenTelemetry Documentation",
+    "out": "../../documentation",
```

Comment:
If the documentation is generated in the package, it should output in the package as well, not at the root.

Reply:
IMO as we are running command (`yarn docs`) at the root level, it makes sense to generate docs there (to avoid confusion). WDYT?

Reply:
The root command is irrelevant when it defers to Lerna, since this means there could be a `docs` command in multiple packages. If that was the case and they all used a folder at the root, it could potentially result in conflicts.

Reply:
Theoretically, will just have a single folder for API documentation and most likely will not add `docs` script in another package to create conflicts. Anyways, I have changed the path to package-specific location (`docs/out`).